src/Chain/Chat/MessageStore/CacheStore.php

```diff
@@ -16,6 +16,9 @@ public function __construct(
         private string $cacheKey,
         private int $ttl = 86400,
     ) {
+        if (!interface_exists(CacheItemPoolInterface::class)) {
+            throw new \RuntimeException('For using the CacheStore as message store, the psr/cache package is required. Try running "composer require psr/cache"');
```

Comment:
I wonder how to phrase this, since just installing `psr/cache` won't help, right? it's just the contract and it would need a concrete implementation, see https://packagist.org/providers/psr/cache-implementation

Reply:
like this? d07eb003b8bd0db3679e673bda32b0143f585a74

Reply:
yup, good one :+1: